microsetta_interface/templates/kits.jinja2

```diff
@@ -149,13 +149,15 @@
             }
         }
 
-        function updateButtonState(kit_id_value) {
-            if(kit_id_value != "") {
+        /*
```

Comment:
should this code be deleted?

Reply:
I'd like to revisit making the button responsive to user input, so I'd prefer not to delete the code entirely.